### PR TITLE
Jitx 6160/fix altium symbol text orientation

### PR DESCRIPTION
**Pull request by lchao-jitx**
Fix the orientation of the text for OCDB Altium power symbols.

See this [comment](https://linear.app/jitx/issue/JITX-6160#comment-db04349d) in the ticket for testcase.

### CODE REVIEW

File: utils/symbols.stanza.SAVE
Comment (bhusang) [inline utils/symbols.stanza.SAVE:1]: Did you mean to check this file in? Probably not necessary and likely cluttering.

**Review by erwin-lau — COMMENTED** [review]
What is the SAVE file?

File: utils/symbols.stanza.SAVE
Comment (lchao-jitx) [reply utils/symbols.stanza.SAVE:1]: Oops. No. That should be removed.

File: utils/symbol-utils.stanza [hunk 1]
```diff
@@ -135,6 +135,12 @@ public defn unit-rectangle (p0:Point|[Double, Double], p1:Point|[Double, Double]
 
 val MM-TO-POINT = 72.0 / 24.5
 
+public defn unit-text (l:String, p:Pose, txt, size:Double, a:Anchor, scale:Double) :
+  inside pcb-symbol :
+    ;val pt-size = to-int(ceil(size * scale * MM-TO-POINT)) [TODO: Fix this]
+    val pt-size = 0.7056 ;[Placeholder]
```
Comment (PhilippeFerreiraDeSousa) [inline utils/symbol-utils.stanza:141]: I see you just copy pasted. Not great to leave such thing in ocdb which is public but it was already so okay

**Review by PhilippeFerreiraDeSousa — APPROVED** [review]

**Review by ariel-hi — APPROVED** [review]
An exciting fix! Nice!

**Pull request merged**